balancer/rls/control_channel_test.go

```diff
@@ -406,8 +406,8 @@ func (s) TestControlChannelCredsFailure(t *testing.T) {
 				DialCreds: clientCreds,
 				Authority: "authority-mismatch",
 			},
-			wantCode: codes.Unavailable,
-			wantErr:  "transport: authentication handshake failed: x509: certificate is valid for *.test.example.com, not authority-mismatch",
+			wantCode:     codes.Unavailable,
+			wantErrRegex: regexp.MustCompile(`transport: authentication handshake failed:.*x509: certificate is valid for \*.test.example.com, not authority-mismatch`),
```

Comment:
This `x509: cert....` stuff is still from stdlib error strings.

I think having the domains/authority in there is pretty valid, since any good error message should contain them, but please remove things that are stdlib error strings.